### PR TITLE
Landing page fixes

### PR DESCRIPTION
**Pull request by gitChimp88**
Fix button disappearing on re-size
Add font awesome can to index.html to show social icons
Fix explosion below main button
Improve gradient transition between apng and bottom half of page

### CODE REVIEW

File: app/src/App.js [hunk 1]
```diff
@@ -78,15 +78,10 @@ function createParticle(x, y, type) {
   let destinationX = (Math.random() - 0.5) * 3000;
   let destinationY = (Math.random() - 0.5) * 3000;
   let rotation = Math.random() * 520;
-  // let delay = Math.random() * 200;
 
-  switch (type) {
-    case 'gesture':
-      particle.innerHTML = String.fromCodePoint(0x1F90C)
```
Comment (nunocgoncalves) [inline app/src/App.js:85]: **Problem:** Seems like some browser don't support emojis using codepoint.
**Proposal:** Let's try and use an image for the emoji effect

File: app/src/index.css [hunk 2]
```diff
@@ -7,7 +7,14 @@
 body {
   margin: 0;
   padding: 0;
-  background: radial-gradient(#051223, #051223, #030e1c);
+  background: radial-gradient(
```
Comment (nunocgoncalves) [inline app/src/index.css:10]: **Problem:** Weird background artifact and color transition
**Proposal:** Make the color shift more progressive in order to match the image better and adjust radius to remove center artifact.
image

This tool might be useful: [CSS Gradient](https://cssgradient.io/)

**Review by nunocgoncalves — CHANGES_REQUESTED** [review]
**Problems:**
The bugs described bellow are related to the mobile view:
- The APNG seems to get cropped
- The gradient rotates 180
image

**Proposal:** Fix image resizing and adjust gradient with media queries

File: app/src/index.css (hunk 2 above)
Comment (gitChimp88) [reply app/src/index.css:10]: As discussed will try a different approach with seperate background and animation

File: app/src/App.js (hunk 1 above)
Comment (gitChimp88) [reply app/src/App.js:85]: Good catch! I've replaced it with an image now

**Review by nunocgoncalves — CHANGES_REQUESTED** [review]
**Bugs:**

- When clicking on the same line as the button, as represented bellow, the button gets activated.
image

**Improvements:**

Neon Sign:
The video animation should be bigger just like on the current version of the website.

Effect:
I like the effect but it needs some optimizations:
1. The emoji should be bigger
2. The effect should generate less emojis
3. Let's try and use a compressed image version of the emoji as the effect is still slow.

**Review by nunocgoncalves — APPROVED** [review]

**Pull request merged**